app/src/main/res/values/strings.xml

```diff
@@ -32,9 +32,9 @@
 
     <!-- Main activity: Action bar -->
     <string name="main_action_bar_title">Subscribed topics</string>
-    <string name="main_menu_notifications_enabled">Notifications enabled</string>
-    <string name="main_menu_notifications_disabled_forever">Notifications disabled</string>
-    <string name="main_menu_notifications_disabled_until">Notifications disabled until %1$s</string>
+    <string name="main_menu_notifications_enabled">Notifications paused</string>
```

Comment:
Typo: This should probably say "resumed".

So in the context of "pause", the word "resume" is fine. But in the context of "it's on" without pausing it it's strange. Arguably it doesn't matter much here since it's hidden behind the icon, but it's still a little strange. Open for suggestions.